index.html

```diff
@@ -145,125 +148,409 @@ <h2>
     </section>
     <section>
       <h2>
-        <code>pointerlockchange</code> and <code>pointerlockerror</code> Events
+        Glossary
       </h2>
-      <p>
-        Two events are used to communicate pointer lock state change or an
-        error in changing state. They are named <dfn data-dfn-for="Document"
-        data-dfn-type="event">pointerlockchange</dfn> and <dfn data-dfn-for=
-        "Document" data-dfn-type="event">pointerlockerror</dfn>. If pointer
-        lock is entered or exited for any reason a
-        {{Document/pointerlockchange}} event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <dl>
+        <dt>
+          <dfn>Engagement gesture</dfn>
```

Comment:
Please drop the "engagement gesture" glossary entry and all references to it.  It was removed from the spec in https://github.com/w3c/pointerlock/pull/76.

The term "transient activation" in the current spec captures the same idea.  In "algorithmic" spec language, this is like "the 'relevant global object' having 'transient activation'...".

Reply:
Agree.

Reply:
Done.